src/app/register-item/components/TagInput.tsx

```
@@ -0,0 +1,115 @@
+          disabled={disabled}
+        />
+      </div>
+      {suggestions.length > 0 && (
+        <div className="mt-2 border rounded-md bg-background">
+          {suggestions.map((suggestion) => (
+            <div
+              key={suggestion}
+              onClick={() => addTag(suggestion)}
+              className="p-2 hover:bg-muted cursor-pointer text-sm"
+            >
+              {suggestion}
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)